Assets/Scripts/AI/Entities/Colonist.cs

```diff
@@ -1,19 +1,44 @@
 using UnityEngine;
+using WorldAPI.Tasks.Generic;
 
-public class Colonist : AIEntity
+public partial class Colonist : ResourceGatherer<Colonist>
 {
-    public void Start()
+    public new void Start()
```

Comment:
What purpose does the new keyword use?

Reply:
Method/Property hiding. It is basically `override` if the base class doesn't implement `virtual` or `abstract`.